config/clusters/2i2c/jackeddy.values.yaml

```diff
@@ -0,0 +1,187 @@
+                  kubespawner_override:
+                    mem_guarantee: 7970807808
+                    mem_limit: 7970807808
+                    cpu_guarantee: 0.9375
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_14_8:
+                  display_name: 14.8 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 15941615616
+                    mem_limit: 15941615616
+                    cpu_guarantee: 1.875
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_29_7:
+                  display_name: 29.7 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 31883231232
+                    mem_limit: 31883231232
+                    cpu_guarantee: 3.75
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_60_6:
+                  display_name: 60.6 GB RAM, upto 15.72 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 65105797120
+                    mem_limit: 65105797120
+                    cpu_guarantee: 7.86
+                    cpu_limit: 15.72
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_121_3:
+                  display_name: 121.3 GB RAM, upto 15.72 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 130211594240
+                    mem_limit: 130211594240
+                    cpu_guarantee: 15.72
+                    cpu_limit: 15.72
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+            image:
+              display_name: Image
+              # https://infrastructure.2i2c.org/howto/features/allow-unlisted-profile-choice/
+              unlisted_choice:
+                enabled: True
+                display_name: "Custom image"
+                validation_regex: "^.+:.+$"
+                validation_message: "Must be a publicly available docker image, of form <image-name>:<tag>"
+                kubespawner_override:
+                  image: "{value}"
+              choices:
+                pangeo:
+                  display_name: Base Pangeo Notebook
+                  default: true
+                  slug: "pangeo"
+                  kubespawner_override:
+                    image: "pangeo/pangeo-notebook:2023.10.03"
+    hub:
+      allowNamedServers: true
+      image:
+        name: quay.io/2i2c/unlisted-choice-experiment
+        tag: "0.0.1-0.dev.git.7130.h0bdc2d30"
+      networkPolicy:
+        # FIXME: For dask gateway
+        enabled: false
+      readinessProbe:
+        enabled: false
```

Comment:
@yuvipanda, this config is copied from https://github.com/2i2c-org/infrastructure/pull/1466.

Do we still need this now, or was it a hack to fix an issue that is not present anymore?

Reply:
We don't need to disable readinessProbe any more, it doesn't spam our logs and doesn't cause us to become unready any more - the livenessProbe causes a restart before the pod doesn't get ready.

We also don't need to disable a networkPolicy for dask-gateway any more.

Reply:
If you clean it up here, I'll open up a PR to clean it up elsewhere.

Reply:
Super! Thank you for the explanation on this one. Didn't know where these were coming from.

Reply:
Here is z2jh defaults - readinessProbe will not cause failures before livenessProbe

```yaml
  livenessProbe:
    # The livenessProbe's aim to give JupyterHub sufficient time to startup but
    # be able to restart if it becomes unresponsive for ~5 min.
    enabled: true
    initialDelaySeconds: 300
    periodSeconds: 10
    failureThreshold: 30
    timeoutSeconds: 3
  readinessProbe:
    # The readinessProbe's aim is to provide a successful startup indication,
    # but following that never become unready before its livenessProbe fail and
    # restarts it if needed. To become unready following startup serves no
    # purpose as there are no other pod to fallback to in our non-HA deployment.
    enabled: true
    initialDelaySeconds: 0
    periodSeconds: 2
    failureThreshold: 1000
    timeoutSeconds: 1
```

Reply:
https://github.com/2i2c-org/infrastructure/pull/3257 opened about this

Reply:
Thank you @consideRatio <3